### DISCUSSION TITLE
Is it possible to refer to other columns in atomic set?

### DISCUSSION THREAD

**Discussion by ronenteva** [prisma/prisma#7260] [Q&A] [answered]
Is it possible to refer to other columns in atomic `set` operation?
If not, what will be the most efficient way to do something like this:

```prisma
model Stats {
  views     Int      @default(0)
  clicks    Int      @default(0)
  ratio     Float    @default(0)
}
```
```ts
prisma.stats.updateMany({
      data: {
        views: {
          increment: 1
        },
        ratio: {
          set: clicks / views
        }
      }
    })
```

I know it can be done easily with generated columns which is not yet supported.

**Comment by ryands17** [comment, marked as answer]
@ronenteva 👋 

Currently not, but there's a feature request for the same: https://github.com/prisma/prisma/issues/5048

It would be great if you could add a 👍 to this so that we can look at the priority. A workaround would be [raw queries](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access).

**Reply by ronenteva** [reply]
Done, thanks!
